app/components/request/configs-trigger-build-notice.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { equal } from '@ember/object/computed';
+import { pluralize } from 'ember-inflector';
+
+export default Component.extend({
+  tagName: 'p',
```

Comment:
We're striving to make all new components tagless, because this concept has recently changed in Ember. In recent updates all Ember components are tagless, which means that `tagName` is always `''` (the property is deprecated and will be removed at some point), no bindings are made in component class, all markup and bindings are moved to template entirely (no wrapper element, template = DOM). The component class itself is not required anymore, by default Ember now generates only templates for components (so called "template-only components").

While old-style components still working properly, it makes sense to lean towards framework direction and use new recommended approaches whenever possible.

Some time soon we're going to update to the latest Ember version and would have to refactor all old-style components anyway.

Reply:
interesting. i assume "tagless" does not mean "does not generate it's own tag", but just "does not produce tags other than `div`"?

done: https://github.com/travis-ci/travis-web/commit/4c3d455381b26d80640dee469b321936ac1e2b88

Reply:
Right, "tagless" in Ember means that this component won't have a wrapper element. In order to make a component "tagless", it's necessary to set `tagName: ''` - to be empty string. Otherwise it'll default to `div` and will create a wrapper `<div>` element.

Here's a little bit more info:
![image](https://user-images.githubusercontent.com/4066489/78701236-5b36a680-790f-11ea-80e1-f31e8e790c89.png)
(posting as a screenshot because it doesn't allow to link to a specific section). Full post is here: https://blog.emberjs.com/2019/12/20/octane-is-here.html

Reply:
In your fix it'll still create a wrapper `div`

Reply:
thanks! 

done: https://github.com/travis-ci/travis-web/pull/2399/commits/97cd0a57cedebb20923b417440636d99809a866b

Reply:
Nice! Thank you @svenfuchs!